playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -60,12 +73,11 @@
   hosts: oo_etcd_to_migrate
   gather_facts: no
   pre_tasks:
-  - set_fact:
-      l_etcd_service: "{{ 'etcd_container' if openshift.common.is_containerized else 'etcd' }}"
-  - name: Disable etcd members
-    service:
-      name: "{{ l_etcd_service }}"
-      state: stopped
+  - include_role:
+      name: etcd
+      tasks_from: disable_etcd
+    vars:
+      r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
```

Comment:
This seems safe but unrelated to the PR description. Just checking to make sure you've tested this and it wasn't an inadvertent update.

Reply:
It's a small refactoring, though surely not related to the bug fix. I tested that.